exams/pearson/upload.py

```diff
@@ -28,6 +28,10 @@ def get_connection():
         if getattr(settings, key) is None:
             missing_settings.append(key)
 
+    pearson_exams_sync = settings.FEATURES.get("PEARSON_EXAMS_SYNC", False)
```

Comment:
@rhysyngsun So in `upload_tsv` when `get_connection()` is None, what exception we should raise exception or just skip?